airbyte-cdk/python/unit_tests/sources/file_based/file_types/test_csv_parser.py

```diff
@@ -501,6 +501,55 @@ def test_given_too_few_values_for_columns_when_read_data_then_raise_exception_an
             next(data_generator)
         assert new_dialect not in csv.list_dialects()
 
+    def test_given_too_many_values_for_columns(self) -> None:
```

Comment:
Nit; would you mind using pytest.parametrized for these new tests instead of the `for` loop?

Reply:
Fixed